src/app/config.py

```diff
@@ -23,3 +23,7 @@
 
 SUPERUSER_LOGIN: str = os.getenv("SUPERUSER_LOGIN")
 SUPERUSER_PWD: str = os.getenv("SUPERUSER_PWD")
```

Comment:
in which case could we not have the credentials ?

Reply:
When the environment variable was not set, just like it happened earlier today on Heroku :upside_down_face: